discord/ext/commands/converter.py

```diff
@@ -784,41 +828,36 @@ async def convert(self, ctx: Context, argument: str) -> str:
         transformations = {}
 
         if self.fix_channel_mentions and ctx.guild:
+
             def resolve_channel(id, *, _get=ctx.guild.get_channel):
                 ch = _get(id)
                 return f'<#{id}>', ('#' + ch.name if ch else '#deleted-channel')
 
             transformations.update(resolve_channel(channel) for channel in message.raw_channel_mentions)
 
         if self.use_nicknames and ctx.guild:
+
             def resolve_member(id, *, _get=ctx.guild.get_member):
                 m = _get(id)
                 return '@' + m.display_name if m else '@deleted-user'
+
         else:
+
             def resolve_member(id, *, _get=ctx.bot.get_user):
                 m = _get(id)
                 return '@' + m.name if m else '@deleted-user'
 
+        transformations.update((f'<@{member_id}>', resolve_member(member_id)) for member_id in message.raw_mentions)
 
-        transformations.update(
-            (f'<@{member_id}>', resolve_member(member_id))
-            for member_id in message.raw_mentions
-        )
-
-        transformations.update(
-            (f'<@!{member_id}>', resolve_member(member_id))
-            for member_id in message.raw_mentions
-        )
+        transformations.update((f'<@!{member_id}>', resolve_member(member_id)) for member_id in message.raw_mentions)
```

Comment:
Please disable auto formatting on these lines, they're written that way for readability.